### PR TITLE
Change the base dir and add suggesting using Pebbledb

### PR DESCRIPTION
**Pull request by huyngopt1994**
Seem we missmatched basedir for starting command in 5th step.

### CODE REVIEW

**Comment by minh-bq** [comment]
Why do we need to change the base directory?

**Comment by huyngopt1994** [comment]
@minh-bq https://github.com/ronin-chain/ronin-documentation/commit/51a86373c6e6e866c1f698844450ad895031d154#diff-6f2bbbd7a44fdbc7e2073d48a3efcf1cb48e9ddc72955bd77b11762dcda32aa1 Seem @RodinGolodin updated the in step 1 for creating sub directory docker but missing for updating in 5th step which leads this error

**Comment by RodinGolodin** [comment]
> @minh-bq [51a8637#diff-6f2bbbd7a44fdbc7e2073d48a3efcf1cb48e9ddc72955bd77b11762dcda32aa1](https://github.com/ronin-chain/ronin-documentation/commit/51a86373c6e6e866c1f698844450ad895031d154#diff-6f2bbbd7a44fdbc7e2073d48a3efcf1cb48e9ddc72955bd77b11762dcda32aa1) Seem @RodinGolodin updated the in step 1 for creating sub directory docker but missing for updating in 5th step which leads this error

Possibly I did miss the update. Thanks for raising this @huyngopt1994 !

**Review by minh-bq — APPROVED** [review]

File: docs/validators/setup/mainnet/run-combined.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ For more information, see [Generate keys](../generate-keys.md).
    LISTENERS__RONIN__STATS__SECRET=WSyDMrhRBe111
    ```
 
-5. (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo:
+5. (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo (Prefer using Pebbledb):
```
Comment (RodinGolodin) [inline docs/validators/setup/mainnet/run-combined.md:217]: @huyngopt1994 Let's rewrite a bit to remove the parentheses:
(Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo, preferably using PebbleDB:

File: docs/validators/setup/mainnet/run-non-validator.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Install Docker Engine and the Docker Compose plugin:
    RONIN_PARAMS=--http.api eth,net,web3,consortium --txpool.pricelimit 20000000000 --txpool.nolocals --cache 4096 --discovery.dns enrtree://AIGOFYDZH6BGVVALVJLRPHSOYJ434MPFVVQFXJDXHW5ZYORPTGKUI@nodes.roninchain.com
    ```
 
-4. (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo:
+4. (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo (Prefer using Pebbledb):
```
Comment (RodinGolodin) [inline docs/validators/setup/mainnet/run-non-validator.md:91]: (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo, preferably using PebbleDB:

File: docs/validators/setup/mainnet/run-validator.md [hunk 3]
```diff
@@ -122,7 +122,7 @@ Generate a private key for your validator node. For more information, see [Gener
    RONIN_PARAMS=--http.api eth,net,web3,consortium --miner.gaslimit 100000000 --miner.gasreserve 10000000 --cache 4096 --discovery.dns enrtree://AIGOFYDZH6BGVVALVJLRPHSOYJ434MPFVVQFXJDXHW5ZYORPTGKUI@nodes.roninchain.com
    ```
 
-4. (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo:
+4. (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo (Prefer using Pebbledb):
```
Comment (RodinGolodin) [inline docs/validators/setup/mainnet/run-validator.md:125]: (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo, preferably using PebbleDB:

**Review by RodinGolodin — CHANGES_REQUESTED** [review]

**Comment by huyngopt1994** [comment]
@RodinGolodin updated

**Review by RodinGolodin — APPROVED** [review]

**Pull request merged**